### PR TITLE
Drv can

### PR DESCRIPTION
**Pull request by RobertoAldea**

### CODE REVIEW

File: code/drv_can/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  # Specify the Python versions you support here. In particular, ensure
+  # that you indicate you support Python 3. These classifiers are *not*
+  # checked by "pip install". See instead "python_requires" below.
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3 :: Only",
```
Comment (mariuscrsn) [inline code/drv_can/pyproject.toml:99]: Remove it

File: code/drv_can/src/can_sniffer/__version__.py
Comment (mariuscrsn) [inline code/drv_can/src/can_sniffer/__version__.py:1]: Not track this file on git

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/drv_can/src/can_sniffer/__version__.py
Comment (RobertoAldea) [reply code/drv_can/src/can_sniffer/__version__.py:1]: Done

File: code/drv_can/pyproject.toml (hunk 1 above)
Comment (RobertoAldea) [reply code/drv_can/pyproject.toml:99]: Done

**Review by mariuscrsn — APPROVED** [review]

**Pull request merged**